org.eclipse.jdt.apt.tests/src/org/eclipse/jdt/apt/tests/RegressionTests.java

```diff
@@ -298,7 +299,7 @@ public void testBugzilla206591B() throws Exception {
 	public void testBugzilla423254() throws Exception {
 		final String projName = RegressionTests.class.getName()
 				+ "423254.Project"; //$NON-NLS-1$
-		IPath projectPath = env.addProject(projName, "1.5"); //$NON-NLS-1$
+		IPath projectPath = env.addProject(projName, "1.8"); //$NON-NLS-1$
```

Comment:
Is this intentional use of 1.8 as opposed to getFirstSupportedJavaVersion ??

Reply:
I believe not, I will double check. On first pass I've mass replaced '1.x' occurrences to '1.8' to quickly check what will be broken, later on I've reviewed all the changes and replaced with the API call where needed.